src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -103,6 +107,11 @@ export class MdProgressSpinner implements OnDestroy {
     this._cleanupIndeterminateAnimation();
   }
 
+  /** Stroke width of the progress spinner. By default uses 10px as stroke width. */
+  @Input()
```

Comment:
I wanted to make a getter/setter because I wanted to avoid that the developer sets the value to `0` or falsy values.